qa/ccs-unavailable-clusters/build.gradle

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+apply plugin: 'elasticsearch.standalone-rest-test'
+apply plugin: 'elasticsearch.rest-test'
+apply plugin: 'elasticsearch.test-with-dependencies'
+
+dependencies {
```

Comment:
I wanted to added this new test to the existing multi-cluster-search qa module, but that one has two clusters and runs yaml tests against both of them. Whenever you add a new IT test to that module it gets run multiple times (one against the remote cluster, one against the ccs cluster). Also this new module uses the rest high level client too. Maybe we should look into merging the two though as a follow-up, I don't like having too many qa modules and they slow down the build.

Reply:
do we have to have a java test for this or can we just use `multi-cluster-search` for it. I mean we can add a 3rd stage where we shutdown one cluster and then run our tests? To me it seems like we can pull it all in yaml land? I can help if you want with that. I am also ok to make it a followup.

Reply:
I am not sure about this. I personally prefer this java part that shuts down a transport as part of the test over gradle magic that shuts clusters down. I spent some time trying to adapt multi-cluster-search but it was complicated in different ways. And this test also has the advantage of using the high-level client when CCS is used, which we don't test otherwise. Can we eventually address this later?